2_deploy_ses_aio/roles/ses-osh/templates/ses-config.yml.j2

```diff
@@ -0,0 +1,28 @@
+ses_cluster_configuration:
+  ses_cluster_name: ceph
+{% if radosgw_keystone %}
+  ses_radosgw_url: "http://{{ hostvars[groups['ses_nodes'][0]].ansible_default_ipv4.address }}:8080/swift/v1"
+{% endif %}
+
+  conf_options:
+      ses_fsid: {{ hostvars[groups['ses_nodes'][0]].conf_options.stdout_lines.0 }}
```

Comment:
This also fails in my local environment where I have installed SES separately and now just running ses and ses-osh roles to configure it. 

Is there some other PR needed for this to work?

Reply:
Yes I think this basically sets the expectations for other roles, and we then need to adapt everywhere else.